src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/SubaccountSupervisor.kt

```diff
@@ -354,13 +354,14 @@ internal class SubaccountSupervisor(
                             OrderStatus.pending -> AnalyticsEvent.TradePlaceOrderStatusPending
                             OrderStatus.untriggered -> AnalyticsEvent.TradePlaceOrderStatusUntriggered
                             OrderStatus.partiallyFilled -> AnalyticsEvent.TradePlaceOrderStatusPartiallyFilled
+                            OrderStatus.partiallyCanceled -> AnalyticsEvent.TradePlaceOrderStatusPartiallyCanceled
                         }
 
                         tracking(orderStatusChangeEvent.rawValue, analyticsPayload)
 
                         when (order.status) {
                             // order reaches final state, can remove / skip further tracking
-                            OrderStatus.cancelled, OrderStatus.filled -> {
+                            OrderStatus.cancelled, OrderStatus.partiallyCanceled, OrderStatus.filled -> {
```

Comment:
I wonder if it's worth making a type of `FinalizedOrderStates` and `PendingOrderStates` to maintain consistency throughout the codebase (it might be a lil difficult cuz we use string literals, but I think even also having `[Finalized/Pending]OrderStateStrings` could be a reasonable in between state